libs/langchain/langchain/output_parsers/openai_functions.py

```diff
@@ -25,8 +25,8 @@ def parse_result(self, result: List[Generation]) -> Any:
             )
         message = generation.message
         try:
-            func_call = message.additional_kwargs["function_call"]
-        except ValueError as exc:
+            func_call = message.additional_kwargs["function_call"].copy()
```

Comment:
should this be deep copied to be safe?

Reply:
Either works i favor shallow copies to keep things light weight though I think it hardly matters in this case